### DISCUSSION TITLE
ScriptReference/Bounds.SqrDistance

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#38] [Notes]
# ScriptReference/Bounds.SqrDistance

https://docs.unity3d.com/ScriptReference/Bounds.SqrDistance.html

**Comment by vertxxyz** [comment]
Similar to [`Bounds.ClosestPoint`](https://docs.unity3d.com/ScriptReference/Bounds.ClosestPoint.html):
> If the point is inside the bounding box, unmodified point position will be returned.

The result is not signed.
